designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/ImageBlock.kt

```diff
@@ -65,7 +67,8 @@ fun <T> ImageBlock(
                 modifier = Modifier
                     .fillMaxWidth()
                     .clip(shape = RoundedCornerShape(Radius.S))
-                    .height(160.dp),
+                    .height(160.dp)
+                    .clickable { onImageClick.invoke() },
```

Comment:
do we need to do anything here with the focus handling?. I mean if we want always a component is pressed to take the focus.

Reply:
good idea I'll modify it.